docs/en/flight_modes/offboard.md

```diff
@@ -90,20 +95,93 @@ Before using offboard mode with ROS 2, please spend a few minutes understanding
+| Speed + Attitude     | velocity          |                                                               | &check;                                                 |                                                               | &check;                                                       |                                                       |                                                               |
+| Speed + Rate         | velocity          |                                                               | &check;                                                 |                                                               | &cross;                                                       | &check;                                               |                                                               |
+| Speed + Steering     | velocity          |                                                               | &check;                                                 |                                                               | &cross;                                                       | &cross;                                               | &check;                                                       |
+| Throttle + Attitude  | attitude          |                                                               |                                                         | &check;                                                       | &check;                                                       |                                                       |                                                               |
+| Throttle + Rate      | body_rate         |                                                               |                                                         | &check;                                                       |                                                               | &check;                                               |                                                               |
+| Throttle + Steering  | thrust_and_torque |                                                               |                                                         | &check;                                                       |                                                               |                                                       | &check;                                                       |
+
+::: info
+If you intend to use the rover setpoints, we recommend using the [PX4 ROS 2 Interface Library](../ros2/px4_ros2_interface_lib.md) instead since it simplifies the publishing of these setpoints.
+:::
+
+#### Actuator Setpoints
+
+The actuators can be directly controlled using [ActuatorMotors](../msg_docs/ActuatorMotors.md) and [ActuatorServos](../msg_docs/ActuatorServos.md).
```

Comment:
```suggestion
Instead of controlling the vehicle using position, speed, rate and other setpoints, you can directly control the motors and actuators using [ActuatorMotors](../msg_docs/ActuatorMotors.md) and [ActuatorServos](../msg_docs/ActuatorServos.md).
```